### PR TITLE
General review of oCIS docs

### PR DESCRIPTION
**Pull request by phil-davis**

### CODE REVIEW

**Review by EParzefall — APPROVED** [review]

File: modules/ROOT/pages/configuration/office-integration.adoc [hunk 1]
```diff
@@ -96,7 +96,7 @@ DEMO_USERS=
 WOPISERVER_DOCKER_TAG=
 # cs3org wopi server domain. Defaults to "wopiserver.owncloud.test"
 WOPISERVER_DOMAIN=
-# JWT secret which is used for the documents to be request by the Wopi client from the cs3org Wopi server. Must be change in order to have a secure Wopi server. Defaults to "LoremIpsum567"
+# JWT secret which is used for the documents to be requested by the Wopi client from the cs3org Wopi server. Must be change in order to have a secure Wopi server. Defaults to "LoremIpsum567"
```
Comment (EParzefall) [inline modules/ROOT/pages/configuration/office-integration.adoc:99]:
```suggestion
# JWT secret which is used for the documents to be requested by the Wopi client from the cs3org Wopi server. Must be changed in order to have a secure Wopi server. Defaults to "LoremIpsum567"
```

File: bin/instructions_makepdf.md [hunk 2]
```diff
@@ -54,29 +54,29 @@ The general idea of `asciidoctor-pdf` is to create a pdf from a given single fil
...
-You can adjust the settings as needed. `:module_base_path:` is the path to the base document files of the buildable manuals and must be set accordingly. The script takes the chosen template file, adds based on the table of contents of the manual (`nav.adoc`) include statements like below
+You can adjust the settings as needed. `:module_base_path:` is the path to the base document files of the buildable manuals and must be set accordingly. The script takes the chosen template file and, based on the table of contents of the manual (`nav.adoc`), adds include statements like below
 ```
 include::{module_base_path}index.adoc[leveloffset=+1]
 
 include::{module_base_path}installing.adoc[leveloffset=+1]
 ...
 ```
-and pipes this as source to the processor. You can define the level how deep you want to render the table of contents with `:toclevels:`. The pdf file created locally is saved in the `pdf_web` directory with a naming defined by the script.
+and pipes this as source to the processor. You can define the how deep you want to render the table of contents with `:toclevels:`. The pdf file created locally is saved in the `pdf_web` directory with a naming defined by the script.
```
Comment (EParzefall) [inline bin/instructions_makepdf.md:64]:
```suggestion
and pipes this as source to the processor. You can define how deep you want to render the table of contents with `:toclevels:`. The pdf file created locally is saved in the `pdf_web` directory with a naming defined by the script.
```

**Review by EParzefall — APPROVED** [review]

**Comment by mmattel** [comment]
Thanks a lot 😃

**Comment by phil-davis** [comment]
> Thanks a lot smiley

No problem. I noticed a couple of small things that had happened in recent PRs, so did a "scan" of the docs.

**Pull request merged**